### DISCUSSION TITLE
Can `turbo gen` support remote generators?

### DISCUSSION THREAD

**Discussion by bingtsingw** [vercel/turborepo#8575] [Ideas]
### Goals

1. make `turbo gen --config` support remote url

### Background

In an org, we may have lots of projects which share similar patterns and generators, it would be helpful if we can maintain them in one place, not in all projects.

### Proposal

`turbo gen --config <path>/<url>`

**Comment by anthonyshew** [comment]
Copy accepts a URL: https://turbo.build/repo/docs/reference/generate#--copy-nameurl

**Reply by bingtsingw** [reply]
> Copy accepts a URL: https://turbo.build/repo/docs/reference/generate#--copy-nameurl

Copy workspace support url, but custom generators do not support remote url

**Reply by anthonyshew** [reply]
Ah, I see. How would you want it to work, from an API design standpoint?
